src/util/Util.js

```diff
@@ -245,7 +245,7 @@ class Util {
     })
       .then(res => {
         if (res.ok) return res.json();
-        throw res;
+        throw new DiscordError('TOKEN_INVALID');
```

Comment:
shouldn't a 5xx error be taken into account here?

Reply:
reviewing the issue again, this seems to always seems to return 401
I suppose keeping `throw res` as fallback might not be a bad idea